devel/mockery/Portfile

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           golang 1.0
+
+go.setup            github.com/vektra/mockery 3.3.2 v
+go.offline_build    no
+revision            0
+
+description         Mock code autogenerator for Go
+
+long_description    mockery provides the ability to easily generate mocks for \
+                    Golang interfaces using the stretchr/testify/mock package. \
+                    It removes the boilerplate coding required to use mocks.
+
+categories          devel
+installs_libs       no
+license             BSD-3-Clause
```

Comment:
This Portfile looks good overall, I just think that this might not be a valid value for `license`.  Here's the list of licenses as per the MacPorts wiki: https://trac.macports.org/wiki/PortfileRecipes#licensekeyword

Reply:
correct, it should just be "BSD"

Reply:
Thanks for the review! License id is fixed now.